_conferences/devfest-santiago-2019.md

```diff
@@ -0,0 +1,10 @@
+---
+name: "GDG Devfest"
+website: https://devfestsantiago.com/
+location: Santiago, Chile
+
+date_start: 2019-10-04
+date_end:   2019-10-05
+
+cfp_end:   2019-08-31
```

Comment:
This needs a start date to show up. You can set it to today or yesterday if it's already open.